index.js

```diff
@@ -1,26 +1,46 @@
 const marked = require('marked');
 const TerminalRenderer = require('marked-terminal');
-const SemanticReleaseError = require('@semantic-release/error');
+const envCi = require('env-ci');
 const getConfig = require('./lib/get-config');
 const getNextVersion = require('./lib/get-next-version');
 const getCommits = require('./lib/get-commits');
 const logger = require('./lib/logger');
 const {gitHead: getGitHead, isGitRepo} = require('./lib/git');
 
 module.exports = async opts => {
+  const {isCi, branch, isPr} = envCi();
+
+  if (!isCi) {
+    logger.log('This run was not triggered in a CI environment, running in dry-run mode.');
```

Comment:
To be more precise, we could say "This run was not triggered in a _known_ CI environment (https://git.io/vbA4g). Running in dry-run mode."

Reply:
and can `opts.dryRun` be already set to true at this point? In that case the condition should maybe be

```js
if (!isCi && !opts.dryRun) {
```

so folks don’t see that message when running with the `dryRun` option explicitly

Reply:
I made both changes